src/MainViewModel.cpp

```diff
@@ -42,7 +42,7 @@ void MainViewModel::removeCode(CodeViewModel *code)
 
 void MainViewModel::load()
 {
-	QFile file(".local/share/harbour-clubcode/database");
+        QFile file("nemo/home/.local/share/harbour-clubcode/database");
```

Comment:
This path `nemo/home` looks suspect to me. Shouldn't it be `/home/nemo`  if anything?

Also, while Sailfish for now always have a default user named nemo, would it not make sense to maybe use `QDir::home()`? :)

Reply:
The app no longer saves the code with the original path. I have tried XDG_DATA_HOME and $HOME, but didn't succeed that way. I'm aware that this is a quick (and dirty) fix (and I allowed it to fix the app fast), (Any points to a clear description what to use will be welcomed.)

Reply:
Yeah, I figured you had tried your fix before submitting it for review :)

However, I can't help but wonder about that path. `nemo/home` should point to a directory relative to the app's current working directory, and not the user's home directory, which I assume was the intention.

In that regard, the path should be just as valid or invalid as the previously directory.

Do you, by any chance, have a folder named `nemo` in your `/home/nemo` folder on your Sailfish device after launching your updated application?

If so, everything would point towards the working directory being the home folder of the current user (nemo).

If that is the case, then it becomes even more of a mystery as to why the previous path did not work. Perhaps something changed in a recent Sailfish update, disallowing applications to write to the `.local` folder? That seems very odd though.